src/mono/sample/wasm/browser-eventpipe/README.md

```diff
@@ -1,10 +1,10 @@
 [see also](../../../wasm/runtime/diagnostics/README.md)
 
-To be able to run this sample you need to build the runtime with `/p:WasmEnablePerfTracing=true` and use Chrome browser
+To be able to run this sample you need to build the runtime with `/p:FeatureWasmThreads=true` and use Chrome browser
```

Comment:
Shouldn't it be `WasmEnableThreads=true`?

Reply:
It should be `/p:MonoWasmBuildVariant=multithread` thanks!